Quadriga/src/main/java/edu/asu/spring/quadriga/service/projectblog/mapper/IProjectBlogMapper.java

```diff
@@ -0,0 +1,19 @@
+package edu.asu.spring.quadriga.service.projectblog.mapper;
+
+import java.util.List;
+
+import edu.asu.spring.quadriga.domain.projectblog.IProjectBlog;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+
+/**
+ * This interface provides method to communicate with DAO and fetch related objects.
+ * 
+ * @author Pawan Mahalle
+ *
+ */
+public interface IProjectBlogMapper {
+
+	public List<IProjectBlog> getProjectBlogListForProject(String projectId)
```

Comment:
Why does your mapper interface have only one method in it? Shouldn't there be 3?